src/components/Search/SearchOutputGrid.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+    <div class="container-fluid">
+        <div class="row">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 8 spaces.](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=4893952)